### DISCUSSION TITLE
Don't understand the challenge of Lesson 4

### DISCUSSION THREAD

**Discussion by arbdev95** [Cyfrin/foundry-full-course-cu#527] [Q&A] [answered]
hi guys,

i m not understanding how to find the PriceGuess to solve the challenge, can someone help me? Thank you

Best Regards

**Comment by shikhar229169** [comment]
- First of all you need is the price feeds address, by which the challenge contract is getting the price feeds answer.
- The price feeds address represent a contract which is already deployed by chainlink which gives you the price feeds answer.
- Note that you can only get the answer only for a particular asset from a price feeds contract address.
- Its like contract at x address gives you the price feeds of eth -> usd, contract at address y will give you price feeds for eth -> btc, and so on and so forth.
- I would suggest you to check out the docs: https://docs.chain.link/data-feeds/using-data-feeds

**Reply by arbdev95** [reply]
So basically to have the PriceGuess I need the price feed,and after that multiply 10¹⁹,am I correct or wrong?

**Reply by shikhar229169** [reply, marked as answer]
Actually when you get the answer from price feeds contract it is already multiplied by 10 to power 8.
So you only need to multiply the answer from price feeds by 10 to power 11.
